### ISSUE TITLE
Warning in build:webpack

### ISSUE THREAD

**Issue by MyIgel** [engelsystem/engelsystem#623]
```
WARNING: We noticed you're using the `useBuiltIns` option without declaring a core-js version. Currently, we assume version 2.x when no version is passed. Since this default version will likely change in future versions of Babel, we recommend explicitly setting the core-js version you are using via the `corejs` option.

You should also be sure that the version you pass to the `corejs` option matches the version specified in your `package.json`'s `dependencies` section. If it doesn't, you need to run one of the following commands:

  npm install --save core-js@2    npm install --save core-js@3
  yarn add core-js@2              yarn add core-js@3
```

**Comment by marudor** [comment]
Fixed by #628

**Comment by MyIgel** [comment]
The [CI](https://chaos.expert/engelsystem/engelsystem/-/jobs/26301) confirmed the fix and reduced image size in vendor.js from 1.52 MiB to 721 KiB and the total nginx image size from 8.96 MiB to 8.82 MiB :tada:
